Makefile

```diff
@@ -36,6 +36,9 @@ teardown-dev: teardown-db teardown-redis teardown-llm-proxy
 # Add users to db
 add-users-to-db:
 	$(CONDA_ACTIVATE) $(PROJECT_NAME); \
+	set -a && \
+        source $(CURDIR)/deployment/docker-compose/.core_backend.env && \
+        set +a && \
```

Comment:
So that we pull DB credentials, admin user credentials, and redis host from the env var file.